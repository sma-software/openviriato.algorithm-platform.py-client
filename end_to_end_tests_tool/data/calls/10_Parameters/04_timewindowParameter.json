{
    "restResource": "/parameters/{key}",
    "urlParameters": [ "timeWindowParameter" ],
    "httpMethod": "GET",
    "requestBody": null,
    "CSharpMethodName": "GetTimeWindowAlgorithmParameter(key)",
    "CSharpMethodParameterTypes": [ "System.String" ],
    "subcategory": "Get Parameters"
}