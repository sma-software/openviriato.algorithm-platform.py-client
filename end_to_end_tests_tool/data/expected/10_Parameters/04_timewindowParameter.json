{
  "fromTime": "2003-01-01T10:20:30",
  "toTime": "2003-01-01T11:00:00"
}